imports/server/mediasoup.ts

```diff
@@ -386,11 +538,17 @@ class SFU {
   }
 
   onTransportCreated(transport: types.Transport) {
+    const transportAppData = transport.appData as TransportAppData;
+
+    // We're not wiring onTransportCreated up to the monitor router, so this
+    // shouldn't happen
+    if (transportAppData.type !== 'webrtc') {
```

Comment:
If it's indicative of a programming error/unreachable state, should this assert or throw instead of just ignoring the event?